ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OAutoML.scala

```diff
@@ -75,26 +68,15 @@ class H2OAutoML(override val uid: String)
       Map()
     }
     val extra = if (algoParameters.nonEmpty) Map("algo_parameters" -> algoParameters) else Map()
-    Map("include_algos" -> determineIncludedAlgos(), "exclude_algos" -> null) ++ extra
+
+    val essentialParameters = getH2OAutoMLBuildModelsParams() - ("include_algos", "exclude_algos")
```

Comment:
NIT: Maybe a small comment here for future readers?

Reply:
np, will add